compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -693,15 +688,28 @@ static void addLowerToLLVMPasses(OpPassManager &modulePassManager,
   if (enableAArch64SME) {
     modulePassManager.addPass(mlir::arm_sme::createVectorLegalizationPass());
     FunctionLikeNest(modulePassManager)
+        .addPredicatedPass(
+            clForceArmStreaming,
+            [] {
+              // Scalable dispatches forced to use streaming mode may not need
+              // ZA (so that is enabled separately for dispatch regions that
+              // need it below).
```

Comment:
Let's clarify a bit. I think that in situation like this it's good to make it clear that there are separate cases by enumerating :)

```suggestion
              // 1. ArmSME - enable Armv9-A streaming mode (i.e. SSVE) without ZA
              // For dispatch regions that contain scalable vectors (when forced to use streaming mode) and which do not require ZA, enable SSVE without ZA.
```

Feel free to re-use or change, but let's add `1.` and `2.` :) (i.e. same comment for the case below)